docs/walk-through/conditionals.md

```diff
@@ -61,18 +61,20 @@ spec:
       image: alpine:3.6
       command: [sh, -c]
       args: ["echo \"it was tails\""]
-  
+
   - name: heads-tails-or-twice-tails
     container:
       image: alpine:3.6
       command: [sh, -c]
       args: ["echo \"it was heads the first flip and tails the second. Or it was two times tails.\""]
 ```
 
-!!! note
-If the parameter value contains quotes, it may invalidate the govaluate expression. To handle parameters with
-quotes, embed an [expr](https://github.com/antonmedv/expr) expression in the conditional. For example:
+!!! Warning "Nested Quotes"
+    If the parameter value contains quotes, it may invalidate the `govaluate` expression.
+    To handle parameters with quotes, embed an [`expr` expression](../variables.md#expression) in the conditional.
+    For example:
 
+<!-- this is supposed to be inside the infobox above, but markdownlint errors when trying to do that and has no in-line ignore yet (https://github.com/markdownlint/markdownlint/issues/16) -->
```

Comment:
I looked at the wrong `markdownlint` here, this is doable per https://github.com/argoproj/argo-workflows/pull/12809#issuecomment-2036237459